### PR TITLE
Tidy up pyfluent Documentation (part1)

### PR DESCRIPTION
**Pull request by prmukherj**
1. Core page has been eliminated.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -148,7 +148,7 @@ jobs:
 
       - name: Retrieve PyFluent version
         run: |
-          echo "::set-output name=PYFLUENT_VERSION::$(python -c "from ansys.fluent.core import __version__; print(__version__)")"
+          echo "PYFLUENT_VERSION=$(python -c "from ansys.fluent.core import __version__; print(__version__)")" >> $GITHUB_OUTPUT
```
Comment (prmukherj) [inline .github/workflows/ci.yml:151]: This has been put in place to handle the warning with "set-output" being deprecated module.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -259,7 +259,7 @@ jobs:
...
```
Comment (prmukherj) [inline .github/workflows/ci.yml:262]: This has been put in place to handle the warning with "set-output" being deprecated module.

File: .github/workflows/nightly-test-run.yml [hunk 3]
```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Retrieve PyFluent version
         run: |
-          echo "::set-output name=PYFLUENT_VERSION::$(python -c "from ansys.fluent.core import __version__; print(__version__)")"
+          echo "PYFLUENT_VERSION=$(python -c "from ansys.fluent.core import __version__; print(__version__)")" >> $GITHUB_OUTPUT
```
Comment (prmukherj) [inline .github/workflows/nightly-test-run.yml:37]: This has been put in place to handle the warning with "set-output" being deprecated module.

File: .github/workflows/tag-iteration-ends.yml [hunk 4]
```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Get last date of iteration
         id: date
-        run: echo "::set-output name=date::$(date -u --date yesterday +'%m%d')"
+        run: echo "name=date=$(date -u --date yesterday +'%m%d')" >> $GITHUB_OUTPUT
```
Comment (prmukherj) [inline .github/workflows/tag-iteration-ends.yml:21]: This has been put in place to handle the warning with "set-output" being deprecated module.

**Review by ajain-work — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**